std_msgs/Header header

jetracer_direct/objects_msg[] objects
string[] recent_objects
